src/gar.lua

```diff
@@ -855,4 +870,48 @@ function gar.instantDelegateWithdrawal(from, gatewayAddress, vaultId, currentTim
 	}
 end
 
+function gar.instantOperatorWithdrawal(from, vaultId, currentTimestamp)
+	local gateway = gar.getGateway(from)
+
+	if gateway == nil then
+		error("Gateway not found")
+	end
+
+	local vault = gateway.vaults[vaultId]
+	if vault == nil then
+		error("Vault not found")
+	end
+
+	if vaultId == from then
+		error("This gateway is leaving and this vault cannot be instantly withdrawn.")
```

Comment:
Was this to spec or did we decide that a leaving gateway can eat the penalties too?

Reply:
Yes - a leaving gateway can eat the penalties only on their stake aove the minimum.  The minimum stake vault is not allowed to be instantly withdrawn, and they must wait the full 90 days.